### DISCUSSION TITLE
It would be nice to add a blacklist to the disposal.

### DISCUSSION THREAD

**Discussion by L1etty** [Zrips/CMI#7368] [Enhancement]
The CMI Dispose custom model data is very good, but I think it would be good to have a blacklist of items.
It would be great to prevent the disposal of certain items.

For example: 
Blacklistitems:
    - diamond
    - Gold

**Comment by FollowWindCrazyDog** [comment]
稍后回复您.

**Reply by mrfloris** [reply]
Stick to english pls
